### PR TITLE
Add a troubleshooting section to Memory Map Analyzer page.

### PR DESCRIPTION
**Pull request by bguedas**

### CODE REVIEW

File: SDK6UserGuide/images/memory_map_analyzer_couldnt_read.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=https://youtrack.cross/agiles/94-402/current?issue=M0090IDE-4871
+HostUrl=https://youtrack.cross/api/files/136-15162?sign=MTczMDA3MzYwMDAwMHwyNC0xNHwxMzYtMTUxNjJ8WC1uTUxsTVFzZHpfY1Z2c3JGc1dTSzg4c2xDN1BhYWNCNEtRdUg3U2VyUQ0K&updated=1729680361664
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/images/memory_map_analyzer_couldnt_read.png:Zone.Identifier:4]: File to remove?

File: SDK6UserGuide/memoryMapAnalyzer.rst [hunk 2]
```diff
@@ -59,6 +59,31 @@ This view is useful in understanding why a symbol is embedded.
...
+
+.. figure:: images/memory_map_analyzer_couldnt_read.png
+   :alt: Detailed view
+   :align: center
+   :scale: 80%
+
+   Error displayed when Memory Map file could not be read
+
+This may happen if the Memory Map Analyzer cannot find a VEE Port to interpret the Memory Map file. To
+workaround this, you can open your Memory Map file with a text editor add the following line::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/memoryMapAnalyzer.rst:76]:
```suggestion
workaround this, you can open your Memory Map file with a text editor and add the following line::
```

File: SDK6UserGuide/memoryMapAnalyzer.rst [hunk 3]
```diff
@@ -59,6 +59,31 @@ This view is useful in understanding why a symbol is embedded.
...
+
+   <property name="jpf.dir" value="/path/to/your/vee"/>
+
+If a line defining the ``jprf.dir`` already exists, check if the value corresponds to a valid path.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/memoryMapAnalyzer.rst:80]:
```suggestion
If a line defining the ``jpf.dir`` already exists, check if the value corresponds to a valid path.
```

File: SDK6UserGuide/memoryMapAnalyzer.rst [hunk 4]
```diff
@@ -59,6 +59,31 @@ This view is useful in understanding why a symbol is embedded.
...
+
+.. note::
+
+   The path must be the the directory of a built VEE Port. If you build a Kernel Application, the
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/memoryMapAnalyzer.rst:84]:
```suggestion
   The path must be the directory of a built VEE Port. If you build a Kernel Application, the
```

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/images/memory_map_analyzer_couldnt_read.png:Zone.Identifier (hunk 1 above)
Comment (bguedas) [reply SDK6UserGuide/images/memory_map_analyzer_couldnt_read.png:Zone.Identifier]: Yep, removd

File: SDK6UserGuide/memoryMapAnalyzer.rst (hunk 2 above)
Comment (bguedas) [reply SDK6UserGuide/memoryMapAnalyzer.rst]: fixed

File: SDK6UserGuide/memoryMapAnalyzer.rst (hunk 3 above)
Comment (bguedas) [reply SDK6UserGuide/memoryMapAnalyzer.rst]: fixed

File: SDK6UserGuide/memoryMapAnalyzer.rst (hunk 4 above)
Comment (bguedas) [reply SDK6UserGuide/memoryMapAnalyzer.rst]: fixed

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**